Sileo/UI/PackageViewController/PackageViewController.swift

```diff
@@ -683,7 +683,7 @@ class PackageViewController: SileoViewController, PackageQueueButtonDataProvider
         let isPurchased = existingPurchased?.contains(package.package) ?? false
 
         if isPurchased {
-            let info = PaymentPackageInfo(dictionary: ["price": "Paid",
+            let info = PaymentPackageInfo(dictionary: ["price": String(localizationKey: "Package_Paid"),
```

Comment:
Does that fix string not showing?

Reply:
When you are in French you keep seeing "Paid" so now you will see the correct translation

Reply:
> 
> 
> When you are in French you keep seeing "Paid" so now you will see the correct translation

This is only in the simulator, but is something that should be changed